feat: adds details about mask for text element (#117)
Basis-Theory/docs: source/includes/elements/_element_mask.md

# Element Mask

```jsx
var phoneNumberElement = BasisTheory.elements.create('text', {
  targetId: 'myPhoneNumberElement',
  mask: ['(', /\d/, /\d/, /\d/, ')', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]
})
```

> Text elements can restrict and fill user input by using the mask attribute. The mask attribute consists of an array

Reviewer: ```suggestion
Text elements can restrict and fill user input by using the `mask` attribute. It consists of an array
```

> of `RegExp` objects and strings, where the `RegExp` objects are used to restrict input and the strings are used to fill input.

Reviewer: ```suggestion
of `RegExp` objects and strings, used to restrict and fill user input, respectively.
```

> The position of the items in the mask array corresponds to the restriction or fill used for that input's position. The

Reviewer: ```suggestion
The position of each item in the mask array corresponds to the restriction or fill used for that input's position. The
```
